dropwizard-api/src/main/java/org/coner/api/request/AddHandicapGroupRequest.java

```diff
@@ -0,0 +1,9 @@
+package org.coner.api.request;
+
+import org.coner.api.entity.HandicapGroup;
+
+/**
+ *
```

Comment:
Are we going to omit javadocs from these? Seems like a blank javadoc might be tricking checkstyle into passing here.